src/Learning/KDDomainKnowledge/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 file(GLOB cppfiles ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
 add_library(KDDomainKnowledge STATIC "${cppfiles}")
 
+if(GENERATE_VIEWS)
+  target_sources(KDDomainKnowledge PRIVATE KDTextFeatureSpec.dd.log)
+endif()
```

Comment:
Why here we don't use the log list function ?

Reply:
Because in this specific case, the dd file is not in the current project but in  `../KWUserInterface/`. Therefore the log list function won't produce the dd.log since it produces starting from the dd files of the same directory.